composer.json

```
@@ -21,12 +21,12 @@
     ],
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "1.12.x-dev",
+        "sylius/sylius": "1.13.x-dev",
         "symfony/flex": "^1.7",
-        "sylius/invoicing-plugin": "dev-main",
-        "sylius/refund-plugin": "2.0.x-dev",
+        "sylius/invoicing-plugin": "dev-add-support-to-sylius-1.13",
+        "sylius/refund-plugin": "dev-add-support-to-sylius-1.13",
```

According to the suggestion above, there is no need for these changes:
```suggestion
        "sylius/invoicing-plugin": "dev-main",
        "sylius/refund-plugin": "2.0.x-dev",
```